docs/userguide/exceptions.rst

```diff
@@ -42,8 +48,99 @@ of a failed task. For example:
            print('Oops! Something really bad happened')
 
 
+Retries
+-------
+
+Retries are one of the simplest and most frequently used methods to add resiliency
+to ``App`` failures. By retrying failed apps, transient failures (eg. machine failure,
+network failure) and intermittent failures within applications can be addressed.
+When ``retries`` are enabled (set to integer > 0), parsl will automatically
+re-launch applications that have failed, until the retry limit is reached.
+This feature will be available in Parsl since `v0.5.0`.
```

Comment:
This feature will be available in Parsl since `v0.5.0` -> This feature will be available starting in Parsl `v0.5.0`